tests/test_petab.py

```diff
@@ -539,3 +539,15 @@ def test_to_files(petab_problem):  # pylint: disable=W0621
         same_nans = parameter_df.isna() == petab_problem.parameter_df.isna()
         assert ((parameter_df == petab_problem.parameter_df) | same_nans) \
             .all().all()
+
+
+def test_load_remote():
+    """Test loading remote files"""
+
+    yaml_url = "https://raw.githubusercontent.com/PEtab-dev/petab_test_suite" \
+               "/master/cases/0001/_0001.yaml"
+    petab_problem = petab.Problem.from_yaml(yaml_url)
+
```

Comment:
can e.g. also check nr of data / parameters, ... to see it is actually what is expected

Reply:
Can, but I wouldn't find it too helpful. What would be a realistic scenario that would be covered by that? Keep it simple? :)